.github/workflows/update-hedera-response-codes.yml

```diff
@@ -0,0 +1,65 @@
+name: Update HederaResponseCodes.sol
+
+on:
+  workflow_dispatch:
+
+jobs:
+  parse-and-update:
```

Comment:
I suggest to make it verbose as the yaml file

parse-and-update-hedera-response-codes or similar

Reply:
Fixed.